internal/api/v1beta1/group.go

```diff
@@ -228,7 +234,16 @@ func (h Handler) UpdateGroup(ctx context.Context, request *shieldv1beta1.UpdateG
 func (h Handler) ListGroupUsers(ctx context.Context, request *shieldv1beta1.ListGroupUsersRequest) (*shieldv1beta1.ListGroupUsersResponse, error) {
 	logger := grpczap.Extract(ctx)
 
-	users, err := h.userService.ListByGroup(ctx, request.Id, group.MemberPermission)
+	if !utils.IsValidUUID(request.GetOrgId()) {
+		return nil, grpcOrgNotFoundErr
+	}
```

Comment:
Do we really need to add these uuid checks in APIs? I was thinking if we want to support passing org/project names in the request as well that work as uuid? Although we don't check and convert org names to uuid at the moment but we can add that conversion if needed.